src/geoparquet.py

```diff
@@ -30,7 +30,8 @@ def __init__(self, filename, tags, schema_metadata=None, row_group_size=100_000)
 
         Args:
             filename: Path to the output Parquet file
-            tags: List of OSM tags to include in the output
+            tags: A list of OSM tags to include in the output.
+                  List elements may be either strings or a tuple of string and a PyArrow schema.
```

Comment:
I think we can keep the code simpler if instead of permitting list elements to be strings, we require that they always be tuples.

Reply:
Yeah, I'm all for slightly less dynamic typing, but wanted to make the initial diff small for easy review.